test/fixtures/templater/jinja_l_metas/010.sql

```diff
@@ -0,0 +1,5 @@
+{% macro test_macro() %}
```

Comment:
Is this intended to render as an empty file? Seems like the blank line at line 4 means it won't be empty.